src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/Duration/simple_duration.jsx

```diff
@@ -40,7 +42,15 @@ const SimpleDuration = ({
                     {...shared_input_props}
                 />
             }
-            { simple_duration_unit !== 't' &&
+            { simple_duration_unit === 'd' &&
+                <DatePickerWrapper
+                    alignment='left'
+                    is_nativepicker={is_nativepicker}
```

Comment:
I think the `is_nativepicker` prop has been removed or where does it come from?

Reply:
i cant see it anywhere in the store. but its used everywhere, so i dont dare to remove this, can i remove this ?

Reply:
i removed it

Reply:
Could you remove it from the parent component too? (DurationWrapper)

Reply:
sure